Worktez/worktez
# fix(labels): 'making labels dynamic'

Signed-off-by: Abhishek Mishra <am1426620@gmail.com>
### Functionality:
to check and Make the labels dynamic everywhere

### Solution:
made the label calls dynamic everywhere and done some other changes 

### Risk level:

### How to test:
step 1: check the labels everywhere in the codebase and
step 2: see in dashboard in S no -1 is showing as backlog.
step 3: also check task type label in task details
